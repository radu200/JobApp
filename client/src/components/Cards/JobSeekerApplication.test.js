import React from 'react';
import { shallow } from 'enzyme';
import JobSeekerApplication from './JobCard'

describe('CandidateCard',() => {
    it('should render correctly ', () => {
        const component = shallow(<JobSeekerApplication />)
        expect(component).toMatchSnapshot();

    })
    
})