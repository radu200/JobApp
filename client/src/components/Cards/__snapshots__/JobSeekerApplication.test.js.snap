// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CandidateCard should render correctly  1`] = `
<JobCard
  classes={
    Object {
      "JobDetails": "JobCard-JobDetails-1",
      "heading": "JobCard-heading-3",
      "media": "JobCard-media-2",
    }
  }
/>
`;
